.env.example

```
@@ -1,2 +1,5 @@
 BACKEND_URL=http://localhost:8000
-NEXT_PUBLIC_GOOGLE_CLIENT_ID=your-google-client-id.apps.googleusercontent.com
\ No newline at end of file
+
+#for guardrails
+GUARDRAILS_URL = http://localhost:8001
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)